installer/dev/tracelogging.h

```diff
@@ -37,31 +37,48 @@ class WindowsAppRuntimeInstaller_TraceLogger final : public wil::TraceLoggingPro
 
 public:
 
-    BEGIN_COMPLIANT_MEASURES_ACTIVITY_CLASS(Install, PDT_ProductAndServicePerformance);
-    void StartActivity(PCWSTR args, UINT32 options)
+    BEGIN_COMPLIANT_CRITICAL_DATA_ACTIVITY_CLASS(Install, PDT_ProductAndServicePerformance);
+    void StartActivity(PCWSTR args, UINT32 options, bool isElevated)
     {
         TraceLoggingClassWriteStart(Install,
             _GENERIC_PARTB_FIELDS_ENABLED,
             TraceLoggingValue(args, "cmdLineArgs"),
-            TraceLoggingValue(options, "options"));
+            TraceLoggingValue(options, "options"),
+            TraceLoggingValue(isElevated, "isElevated"));
     }
     void StopWithResult(
         HRESULT hresult,
-        UINT32 installStage,
+        UINT32 failureType,
+        PCSTR failureFile,
+        UINT32 failureLineNumber,
+        PCWSTR failureMessage,
+        UINT32 failedInstallStage,
         PCWSTR currentResourceId,
         HRESULT deploymentErrorExtendedHResult,
         PCWSTR deploymentErrorText,
         GUID deploymentErrorActivityId)
     {
         SetStopResult(hresult);
 
-        TraceLoggingClassWriteStop(Install,
-            _GENERIC_PARTB_FIELDS_ENABLED,
-            TraceLoggingValue(installStage, "FailedInstallStage"),
-            TraceLoggingValue(currentResourceId, "CurrentResourceId"),
-            TraceLoggingValue(deploymentErrorExtendedHResult, "DeploymentErrorExtendedHResult"),
-            TraceLoggingValue(deploymentErrorText, "DeploymentErrorText"),
-            TraceLoggingValue(deploymentErrorActivityId, "DeploymentErrorActivityId"));
+        if (!hresult)
```

Comment:
It will be hresult finally. For privacy approval, just forced a failure event for a success case to get failure event json.